bootstrap_cfn/config.py

```diff
@@ -91,13 +97,170 @@ def process(self):
+            InstanceTenancy="default",
+            EnableDnsSupport="true",
+            CidrBlock=FindInMap("SubnetConfig", "VPC", "CIDR"),
+            EnableDnsHostnames="true",
+        )
+
+        subnet_a = Subnet(
+            "SubnetA",
+            VpcId=Ref(vpc),
+            AvailabilityZone="eu-west-1a",
+            CidrBlock=FindInMap("SubnetConfig", "VPC", "SubnetA"),
+            Tags=Tags(
+                Application=Ref("AWS::StackId"),
+                Network="Public",
+            ),
+        )
+
+        subnet_b = Subnet(
+            "SubnetB",
+            VpcId=Ref(vpc),
+            AvailabilityZone="eu-west-1b",
+            CidrBlock=FindInMap("SubnetConfig", "VPC", "SubnetB"),
+            Tags=Tags(
+                Application=Ref("AWS::StackId"),
+                Network="Public",
+            ),
+        )
+
+        subnet_c = Subnet(
+            "SubnetC",
+            VpcId=Ref(vpc),
+            AvailabilityZone="eu-west-1c",
+            CidrBlock=FindInMap("SubnetConfig", "VPC", "SubnetC"),
+            Tags=Tags(
+                Application=Ref("AWS::StackId"),
+                Network="Public",
+            ),
+        )
+
+        igw = InternetGateway(
+            "InternetGateway",
+            Tags=Tags(
+                Application=Ref("AWS::StackId"),
+                Network="Public",
+            ),
+        )
+
+        gw_attachment = VPCGatewayAttachment(
+            "AttachGateway",
+            VpcId=Ref(vpc),
+            InternetGatewayId=Ref(igw),
+        )
+
+        route_table = RouteTable(
+            "PublicRouteTable",
+            VpcId=Ref(vpc),
+            Tags=Tags(
+                Application=Ref("AWS::StackId"),
+                Network="Public",
+            ),
+        )
+
+        public_route = Route(
+            "PublicRoute",
+            GatewayId=Ref(igw),
+            DestinationCidrBlock="0.0.0.0/0",
+            RouteTableId=Ref(route_table),
+            DependsOn=gw_attachment.title
+        )
+
+        subnet_a_route_assoc = SubnetRouteTableAssociation(
+            "SubnetRouteTableAssociationA",
+            SubnetId=Ref(subnet_a),
+            RouteTableId=Ref(route_table),
+        )
+
+        subnet_b_route_assoc = SubnetRouteTableAssociation(
+            "SubnetRouteTableAssociationB",
+            SubnetId=Ref(subnet_b),
+            RouteTableId=Ref(route_table),
+        )
+
+        subnet_c_route_assoc = SubnetRouteTableAssociation(
+            "SubnetRouteTableAssociationC",
+            SubnetId=Ref(subnet_c),
+            RouteTableId=Ref(route_table),
+        )
+
+        resources = [vpc, subnet_a, subnet_b, subnet_c, igw, gw_attachment,
+                     public_route, route_table, subnet_a_route_assoc,
+                     subnet_b_route_assoc, subnet_c_route_assoc]
+
+        # Hack until we return troposphere objects directly
+        return json.loads(json.dumps(dict((r.title, r) for r in resources), cls=awsencode))
```

Comment:
Yeah, this is to keep the function implementation exactly the same and to minimize the changes as much as possible in this PR - once we've merged this then we can change each of these functions to either return troposphere resources directly or take a Template object and add resources.